### PR TITLE
feat: seed test identifier

### PR DESCRIPTION
**Pull request by hoa-ngo-gs**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This feature automates the process of seeding a predefined DID document for use in development environments.

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 [vc-kit doc site](https://uncefact.github.io/vckit/)
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: development/did-web-identifier [hunk 1]
```diff
@@ -0,0 +1 @@
+{"did":"did:web:uncefact.github.io:project-vckit:test-and-development","alias":"uncefact.github.io:project-vckit:test-and-development","provider":"did:web","controllerKeyId":"bd3a7c525d86c18fa91f2a8f0354231b85aa0f7f269cc2a7914b6cca4087f0ae","keys":[{"kid":"bd3a7c525d86c18fa91f2a8f0354231b85aa0f7f269cc2a7914b6cca4087f0ae","kms":"local","type":"Ed25519","publicKeyHex":"bd3a7c525d86c18fa91f2a8f0354231b85aa0f7f269cc2a7914b6cca4087f0ae","privateKeyHex":"d10c5893361a0c15aa002387a6f608088dff9ef8e772307c5d06534fb391505b","meta":{"algorithms":["EdDSA","Ed25519"]}}]}
```
Comment (ldhyen99) [inline development/did-web-identifier:1]: is this the hardcode data? That means we only support Ed25519 (not for Secp256r1, Secp256k1, or X25519).

File: documentation/static/test-and-development/did.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:web:uncefact.github.io:project-vckit:test-and-development",
+  "verificationMethod": [
```
Comment (ldhyen99) [inline documentation/static/test-and-development/did.json:7]: To verify Enveloping Proof, we need to type `JsonWebKey` following W3C document [here](https://www.w3.org/TR/vc-jose-cose/#using-controller-documents). Do you think we should update it?
image

File: development/did-web-identifier (hunk 1 above)
Comment (ashleythedeveloper) [reply development/did-web-identifier:1]: @ldhyen99, what crypto suite must we use for enveloping proof (JOSE)?

Comment (ashleythedeveloper) [reply development/did-web-identifier:1]: Answered below..

**Review by ldhyen99 — DISMISSED** [review]

File: development/did-web-identifier
[the remark this answers is not in the document]
Comment (hoa-ngo-gs) [reply development/did-web-identifier:1]: We just need the content. The file format is not important. Worth noting is that there's no whitespace in this file.

**Review by huynguyen-hl — DISMISSED** [review]

File: development/did-web-identifier
[the remark this answers is not in the document]
Comment (namhoang1604) [reply development/did-web-identifier:1]: The file format will make it clear, we can compact the json to clear the whitespace.

**Review by huynguyen-hl — APPROVED** [review]

**Review by namhoang1604 — APPROVED** [review]

**Pull request merged**